src/helpers/resourceIconExtensionMapping.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Do we want to export this file as part of our `dist` bundle?

Reply:
Would be similiar to the translations in https://github.com/owncloud/owncloud-design-system/commit/770d9927f18ba6126a551aeffa0dec92ab1af7a8 (webpack copy plugin)